### ISSUE TITLE
Update Autocomplete to WAI-ARIA 1.2

### ISSUE THREAD

**Issue by bobwise** [sparkdesignsystem/Backlog#1367]
**This task should not be done until the 1.2 version is officially recommended for adoption by the W3C.**

Update `role="combobox"` components to use the WAI-ARIA 1.2 specification. This will likely be a change to the HTML structure of the component and possibly the JavaScript used in the component, which is a breaking change for Spark users.

[Recommended 1.1 version as of 2020](https://www.w3.org/TR/wai-aria-1.1/#combobox)
[Draft 1.2 version as of 2020](https://www.w3.org/TR/wai-aria-1.2/#combobox)

Affects the Autocomplete component.

The 1.2 change is a "breaking change" in the ARIA spec. This means that once 1.2 becomes the recommended version, Spark will fail validation (in axe, etc) until we upgrade.
![image](https://user-images.githubusercontent.com/1424560/103419247-369af280-4b60-11eb-97ed-5ca4dfc96306.png)

**Comment by ssabo1** [comment]
Awareness Issue: will need to update once ready. Awaiting: Aria 1.2

**Comment by ssabo1** [comment]
Confirmed: This has been added to Spark Design System Team Backlog.
